azkaban-common/src/test/java/azkaban/ServiceProviderTest.java

```diff
@@ -71,7 +76,34 @@ public void testInjections() throws Exception {
     assertThat(injector.getInstance(StorageManager.class)).isNotNull();
     assertThat(injector.getInstance(DatabaseStorage.class)).isNotNull();
     assertThat(injector.getInstance(LocalStorage.class)).isNotNull();
+    assertThatThrownBy(
```

Comment:
Is this assert necessary?

Reply:
The ideal test should test out if runtime needs to resolve hadoop jars. However, it cannot be straightforwardly tested. This test can guarantee that Hadoop module is not installed, so I think it has values.